test/gtest/ucp/test_ucp_am.cc

```diff
@@ -1019,6 +1019,11 @@ UCS_TEST_P(test_ucp_am_nbx_eager_data_release, multi_bcopy, "ZCOPY_THRESH=inf")
     test_data_release(fragment_size() * 2);
 }
 
+UCS_TEST_P(test_ucp_am_nbx_eager_data_release, multi_zcopy, "ZCOPY_THRESH=0")
+{
+    test_data_release(UCS_MBYTE);
```

Comment:
better to use fraction of fragment size to be sure it is indeed multi-fragment

Reply:
It was:
```
UCS_TEST_P(test_ucp_am_nbx_eager_data_release, multi_zcopy, "ZCOPY_THRESH=0")
{
    if (zcopy_fragment_size() == 0) {
        UCS_TEST_SKIP_R("zcopy is not supported");
    }
    test_data_release(zcopy_fragment_size() * 2);
}
```
It was changed due to https://github.com/openucx/ucx/pull/7988#discussion_r820894643